content/api/commands/origin.md

```diff
@@ -233,11 +234,42 @@ and the protocol defaults to `https`. When `cy.visit()` is called with the path
 `/history/founder`, the three are concatenated to make
 `https://www.acme.com/history/founder`.
 
+#### Alternative navigation
+
+```js
+// Do things in primary origin...
+
+cy.visit('https://www.acme.com/history/founder')
+
+// The cy.origin block is required to interact with the cross-origin page.
+cy.origin('www.acme.com', () => {
+  cy.get('h1').contains('About our Founder, Marvin Acme')
+})
+```
+
+Here the cross-origin page is visited prior to the `cy.origin` block, but any
+interactions with the window are performed within the block which can
+communicate with the cross-origin page
+
+#### Incorrect usage
```

Comment:
```suggestion
#### **<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
```

Should this have the red exclamation triangle?

Reply:
LGTM:

![Screen Shot 2022-09-27 at 12 21 33 PM](https://user-images.githubusercontent.com/1157043/192581416-985aab0c-2926-47c7-b4c7-d72dd734abed.png)

Addressed in https://github.com/cypress-io/cypress-documentation/pull/4747/commits/59809e4535f7b28141413884c0c21767bffeed94